internal/globaldefault/default.go

```diff
@@ -67,14 +69,19 @@ func SetupDefaultActivation(subshell subshell.SubShell, cfg DefaultConfigurer, r
 		return locale.WrapError(err, "err_globaldefault_rtexes", "Could not retrieve runtime executables")
 	}
 
-	fw := executor.NewWithBinPath(projectPath, BinDir())
+	projectDir := filepath.Dir(proj.Source().Path())
+	fw := executor.NewWithBinPath(projectDir, BinDir())
 	if err := fw.Update(exes); err != nil {
 		return locale.WrapError(err, "err_globaldefault_fw", "Could not set up forwarders")
 	}
 
-	if err := cfg.Set(constants.GlobalDefaultPrefname, projectPath); err != nil {
+	if err := cfg.Set(constants.GlobalDefaultPrefname, projectDir); err != nil {
 		return locale.WrapError(err, "err_set_default_config", "Could not set default project in config file")
 	}
 
+	if err := cfg.Set(constants.ActiveProjectConfig, proj.Namespace().String()); err != nil {
```

Comment:
hmm, do we need both? Technically the project dir could be used to receive the namespace.

We should maybe at least ensure that they match in terms of their pattern, eg. `projects.active.path` and `projects.active.namespace`.

Reply:
What do we intend to use `projects.active.namespace` for? There isn't currently one in the code.

The project path only has the project name. We would have to parse the projectfile in order to get the namespace, but it's certainly feasible.

Reply:
I meant we should replace the value used under `constants.GlobalDefaultPrefname` with `projects.active.namespace`. But only if we actually need it. I still think we might be able to just get rid of it and rely solely on the path.

Reply:
I don't see a clear use case for having the namespace in the config at the moment so I've removed it in favor of the project path which we are already using.